model/metrics/vcs.yaml

```diff
@@ -0,0 +1,150 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: gauge
+    unit: "{change}"
+    stability: experimental
+    #extends: metric_attributes.vcs
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_open
+    type: metric
+    metric_name: vcs.repository.change.time_open
+    brief: 'The amount of time a change (pull request) has been open'
+    instrument: gauge
```

Comment:
it should be a histogram

Reply:
@lmolkova - would it that there would be two histogram buckets based on state? ie. a bucket for opened pre, and a bucket for closed?

Reply:
histograms have buckets per combination of attribute values, i.e. you can see percentiles per state.

Reply:
I see. I'd be fine with this being a histogram bucket.

Reply:
After discussion in the CICD working group, we think this would be best served by a histogram and will make that change pending any objections.

Reply:
In 5a9ffe7ef997110472448ddf519289cd0a34a85f I have combined `vcs.repository.change.time_open` and `vcs.repository.change.time_to_merge` to be `instrument: histogram` in metric `vcs.repository.change.time`.